### PR TITLE
Heading fragment support

### LINKED ISSUE #1356: Add fragment support within headings container
**Is your feature request related to a problem? Please describe.**
Currently, the Headings container directly uses the "text" value in the model meaning that it has no knowledge of Fragments. 

https://github.com/bbc/simorgh/blob/a199761e19a93bb715a35c5960d49b38c4abb0eb/src/app/containers/Headings/index.jsx#L4 https://github.com/bbc/simorgh/blob/a199761e19a93bb715a35c5960d49b38c4abb0eb/src/app/helpers/blockHandlers.js#L9-L10

**Describe the solution you'd like**
The headings container should re-use the logic from the Text container which understands about Paragraph and Fragment. Then the component should be rendered using the correct Headings component EG: Headline. 

An example of rendering the output of one component using another component is in the InlineLink container:
https://github.com/bbc/simorgh/blob/a199761e19a93bb715a35c5960d49b38c4abb0eb/src/app/containers/InlineLink/index.jsx#L10

**Describe alternatives you've considered**
We could add logic to dig down to the Fragments and then implement a shared way of handling the different attributes rather than re-using the Text container logic. 

**Testing notes**
[Tester to complete]

Dev insight: We will need snapshot tests that contain examples of rich text. Should we have an example for Cypress to render rich text in a Headline or Subheading? 

**Additional context**
Italics and bold may end up rendering as the outcome of this and therefore make https://github.com/bbc/simorgh/issues/1276 obsolete 

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #1356

**Overall change:** Enables bold and italic for Headline and Subheadline

**Code changes:**

- Enables use of fragments
- Updates snapshots
- Adds unit tests to capture fragment output

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [x] Test engineer approval
- [x] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: src/app/lib/__snapshots__/globalStyles.test.js.snap [hunk 1]
```diff
@@ -33,7 +33,7 @@ Array [
...
...line-height:1.15;margin:0;}button,input{overflow:visible;}button,select{text-transform:none;}button,[type=\\"button\\"],[type=\\"reset\\"],[type=\\"submit\\"]{-webkit-appearance:button;}button::-moz-focus-inner,[type=\\"button\\"]::-moz-focus-inner,[type=\\"reset\\"]::-moz-focus-inner,[type=\\"submit\\"]::-moz-focus-inner{border-style:none;padding:0;}button:-moz-focusring,[type=\\"button\\"]:-moz-focusring,[type=\\"reset\\"]:-moz-focusring,[type=\\"submit\\"]:-moz-focusring{outline:1px dotted ButtonText;}fieldset{padding:0.35em 0.75em 0.625em;}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal;}progress{vertical-align:baseline;}textarea{overflow:auto;}[type=\\"checkbox\\"],[type=\\"radio\\"]{box-sizing:border-box;padding:0;}[type=\\"number\\"]::-webkit-inner-spin-button,[type=\\"number\\"]::-webkit-outer-spin-button{height:auto;}[type=\\"search\\"]{-webkit-appearance:textfield;outline-offset:-2px;}[type=\\"search\\"]::-webkit-search-decoration{-webkit-appearance:none;}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit;}details{display:block;}summary{display:list-item;}template{display:none;}[hidden]{display:none;}",
```
Comment (pjlee11) [inline src/app/lib/__snapshots__/globalStyles.test.js.snap]: Any idea why `main` is being added here based on this PR?

Comment (sareh) [reply src/app/lib/__snapshots__/globalStyles.test.js.snap]: I believe due to a patch update in the `styled-normalize` package that is pulled through when doing a fresh install.

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Review by dr3 — DISMISSED** [review]

**Comment by sareh** [comment]
Travis keeps raising the error `Error: Protocol error (Page.setLifecycleEventsEnabled): 'Page.setLifecycleEventsEnabled' wasn't found` when running the application for the lighthouse tests.  This branch seems to be the only one with this error. I'm debugging this issue. Everything works fine locally.

**Review by pjlee11 — DISMISSED** [review]
LGTM 👍

**Review by dr3 — DISMISSED** [review]

**Review by pjlee11 — APPROVED** [review]

**Review by dr3 — APPROVED** [review]

**Pull request merged**